awx/ui_next/src/screens/Setting/MiscSystem/MiscSystemDetail/MiscSystemDetail.test.jsx

```diff
@@ -110,6 +121,74 @@ describe('<MiscSystemDetail />', () => {
     assertDetail(wrapper, 'Red Hat customer username', 'mock name');
     assertDetail(wrapper, 'Refresh Token Expiration', '3 seconds');
     assertVariableDetail(wrapper, 'Remote Host Headers', '[]');
+    assertDetail(wrapper, 'Global default execution environment', 'Foo');
+  });
+
+  test('should render execution environment as not configured', async () => {
+    ExecutionEnvironmentsAPI.readDetail.mockResolvedValue({
+      data: {},
+    });
+    let newWrapper;
+    await act(async () => {
+      newWrapper = mountWithContexts(
+        <SettingsProvider
+          value={{
+            ...mockAllOptions.actions,
+            DEFAULT_EXECUTION_ENVIRONMENT: null,
+          }}
+        >
+          <MiscSystemDetail />
+        </SettingsProvider>
+      );
+    });
+    await waitForElement(newWrapper, 'ContentLoading', el => el.length === 0);
+
+    assertDetail(newWrapper, 'Access Token Expiration', '1 seconds');
```

Comment:
We could probably remove some of these detail label and value assertions since we check them in the previous test.